test/NFT/StableUnitDAOogNFTVotes-test.ts

```diff
@@ -0,0 +1,107 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+describe("StableUnitDAO ogNFTVotes", () => {
+    it("should mint new NFT, send old to DEAD address", async () => {
+        const accounts = await ethers.getSigners();
+        const owner = accounts[0];
+
+        const NftMock = await ethers.getContractFactory("NftMock");
+        const nftMock = await NftMock.deploy("NFT mock", "mNFT");
```

Comment:
please stay away from same variable names that different only in register. Feel free to use other tests in repo as code references https://github.com/StableUnit/stableunit_dao/blob/main/test/TimelockVault-test.ts

Reply:
I did use other tests in the repo as reference when making this choice, such as this one:
https://github.com/StableUnit/stableunit_dao/blob/8d7d5d47a259372df2992d66a6bb6365392365bb/test/TimelockVault-test.ts#L26

Nonetheless, will change